### ISSUE TITLE
DateTime serialization contains additional '-' character.

### ISSUE THREAD

**Issue by brianengel** [neuecc/Utf8Json#6]
It looks like the DateTime serialization to ISO8601 contains an additonal hyphen (-) character.

If I had to guess the formatter is producing a "-5" instead of "5" for the timezone value.

```c#
using System;
using Utf8Json;

namespace ConsoleApp6
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            var item = new
            {
                Date = DateTime.Now
            };

            var result = JsonSerializer.ToJsonString(item);

            // Produced Output: {"Date":"2017-09-27T14:31:17.1876168-0-5:00"}
            // Correct Output:  {"Date":"2017-09-27T14:31:17.1876168-05:00"}
        }
    }
}
```

**Comment by Tornhoof** [comment]
I can't reproduce your error
My output is:
{"Date":"2017-09-27T22:48:45.1747779+01:00"}
but it might be related to your negative TZ diff.

**Comment by neuecc** [comment]
sorry and thanks, I've released `v1.0.3`.
